src/main/resources/application.yaml

```diff
@@ -94,3 +94,8 @@ scheduling:
       cron: 0 0 6 * * *
     scan:
       delay: ${TASK_SCAN_DELAY} # In milliseconds
+
+
+proxy:
+  host-name: ${PROXY_HOST_NAME:proxyout.reform.hmcts.net}
+  port: ${PROXY_PORT:8080}
```

Comment:
perhaps env vars are not needed

Reply:
I added for future use if we need in product chart or for other env..

Reply:
if need to override it'll be bootstrapped then. giving the current setup of our apps